docs/source/reference/numpysupported.rst

```diff
@@ -620,6 +620,8 @@ The following :py:class:`Generator` methods are supported:
 * :func:`numpy.random.Generator().f()` (*)
 * :func:`numpy.random.Generator().gamma()` (*)
 * :func:`numpy.random.Generator().geometric()` (*)
+* :func:`numpy.random.Generator().integers()`: Both `low` and `high` are required
+ arguments. Array values for low and high are currently not supported.
```

Comment:
The reason I enforced this was because otherwise, `high` would be a `OptionalType[int]` argument by default that just refused to be cast to `int` of any type down the line when required in the algorithms.